.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+<!--
+Thank you for sending the PR! We appreciate you spending the time to work on these changes.
+
+Help us understand your motivation by explaining why you decided to make this change.
+
+You can learn more about contributing to appwrite here: https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md
+
+Happy contributing!
+
+-->
+
+## Motivation
+
+(Write your motivation here.)
+
+### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
```

Comment:
You could move the contributing guidelines as the last item in this list. We would want the contributor to talk about the PR first.